package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "openwhisk-vscode-extension",
-	"displayName": "vscode extension for Apache Openwhisk",
-	"description": "vscode extension for Apache OpenWhisk",
+	"displayName": "VSCode extension for Apache Openwhisk",
```

Comment:
I think the name should be title case.
So `extension` should be `Extension` I think?
Sorry for the duplicate review.